code/modules/pai/pai.dm

```diff
@@ -169,6 +169,9 @@
 /mob/living/silicon/pai/examine(mob/user)
 	. = ..()
 	. += "Its master ID string seems to be [(!master_name || emagged) ? "empty" : master_name]."
+	//SKYRAT EDIT ADDITION BEGIN - CUSTOMIZATION
+	. += get_silicon_flavortext()
+	//SKYRAT EDIT ADDITION END
```

Comment:
I'm not sure about this one, because that's starting to be a lot of different uses for silicon flavor-text.

Someone that plays as a borg often doesn't necessarily always want their borg examine text to be displayed on their pAI.

Something like this should be put in a different PR, possibly with its own flavor text pref.

Reply:
This is done with assumption that people might use different characters for different applications.
Not gonna lie, you're not wrong that this should be its own thing. I'll probably do such later.